src/utils/identity.js

```diff
@@ -0,0 +1,3 @@
+export default function identity(value) {
+  return value;
+}
```

Comment:
As an aside: identity would then be something like:

`export const identity = x => x;` which is just :sunglasses: